app/src/main/java/org/mifos/mobilebanking/ui/activities/HomeActivity.java

```diff
@@ -139,6 +139,7 @@ protected void onPause() {
     @Override
     protected void onResume() {
         super.onResume();
+        setNavigationViewSelectedItem(R.id.item_home);
```

Comment:
@shiv07tiwari The changes you have made are incorrect. You cannot always change the selected Navigation Item to `Home` on `onResume()` method. Checkout Activity lifecycle, `onResume()` method is called many times.

A simple scenario whereas you solution will fail will be Go to `Charges > Open Another App > Now open Self-Service App > Charges Fragment` will be displayed and Home Navigation will be selected.